posix-configs/eagle/flight/px4.config

```diff
@@ -8,6 +8,7 @@ df_mpu9250_wrapper start
 df_bmp280_wrapper start
 df_hmc5883_wrapper start
 df_trone_wrapper start
+#df_isl_wrapper start
```

Comment:
Could we enable it by default and it just exits if it doesn't find something with a timeout?